src/models/User.js

```diff
@@ -57,6 +57,106 @@ export default (sequelize, DataTypes) => {
         }
       },
     },
+    gender: {
+      type: DataTypes.STRING,
+      validate: {
+        len: {
+          args: [4],
+          msg: 'Gender must be at least 4 characters long.'
+        },
+      }
+    },
+    birthdate: {
+      type: DataTypes.STRING,
```

Comment:
In the user-migration file, you set `birthdate` to have a datatype of DATE, but in here you set it as `STRING`, please use the appropriate one.